public/manifest.json

```diff
@@ -31,16 +31,10 @@
   "background": {
     "scripts": [
       "vendor/async.min.js",
-      "vendor/jQuery.js",
-      "vendor/lodash.js",
-      "vendor/faker.js",
-      "booking_generator.js",
       "background.js"
-    ],
-    "persistent": false
```

Comment:
removing this as to help with the background page inactivity that causes the extension to slow down